not mergable and also build fail looks real:
```
======================================================================
ERROR: test_simple_post_with_cached_timeout_bad_http_response (corehq.apps.repeaters.tests.test_simple_post.SimplePostCacheTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/corehq/apps/repeaters/tests/test_simple_post.py", line 58, in test_simple_post_with_cached_timeout_bad_http_response
    simple_post_with_cached_timeout('abc', 'http://google.com')
  File "/mnt/corehq/apps/repeaters/models.py", line 77, in simple_post_with_cached_timeout
    message = u'{}: {}'.format(resp.status_code, resp.reason)
AttributeError: 'MockResponse' object has no attribute 'reason'
```

--- a/corehq/apps/repeaters/models.py
+++ b/corehq/apps/repeaters/models.py
@@ -69,7 +69,7 @@ def simple_post_with_cached_timeout(data, url, expiry=60 * 60, force_send=False,
         raise RequestConnectionError(e.message)
 
     if not 200 <= resp.status_code < 300:
-        message = 'HTTP response not a 200 or 300'
+        message = u'{}: {}'.format(resp.status_code, resp.reason)
         cache.set(key, message, expiry)
         raise RequestConnectionError(message)
     return resp

--- a/corehq/apps/repeaters/models.py
+++ b/corehq/apps/repeaters/models.py
@@ -69,7 +69,7 @@ def simple_post_with_cached_timeout(data, url, expiry=60 * 60, force_send=False,
         raise RequestConnectionError(e.message)
 
     if not 200 <= resp.status_code < 300:
-        message = u'{}: {}'.format(resp.status_code, resp.reason)
+        message = u'Status Code {}: {}'.format(resp.status_code, resp.reason)
         cache.set(key, message, expiry)
         raise RequestConnectionError(message)
     return resp

--- a/corehq/apps/repeaters/tests/test_simple_post.py
+++ b/corehq/apps/repeaters/tests/test_simple_post.py
@@ -8,7 +8,7 @@ from django.test import SimpleTestCase
 from corehq.apps.repeaters.models import simple_post_with_cached_timeout
 from corehq.apps.repeaters.exceptions import RequestConnectionError
 
-MockResponse = namedtuple('MockResponse', 'status_code')
+MockResponse = namedtuple('MockResponse', 'status_code reason')
 
 
 class SimplePostCacheTest(SimpleTestCase):
@@ -22,7 +22,7 @@ class SimplePostCacheTest(SimpleTestCase):
     def test_simple_post_with_cached_timeout_success(self):
         with patch(
                 'corehq.apps.repeaters.models.simple_post',
-                side_effect=[MockResponse(status_code=200)]) as mock_post:
+                side_effect=[MockResponse(status_code=200, reason='No reason')]) as mock_post:
 
             resp = simple_post_with_cached_timeout('abc', 'http://google.com')
 
@@ -52,7 +52,7 @@ class SimplePostCacheTest(SimpleTestCase):
         """
         with patch(
                 'corehq.apps.repeaters.models.simple_post',
-                side_effect=[MockResponse(status_code=400)]) as mock_post:
+                side_effect=[MockResponse(status_code=400, reason='Ugly')]) as mock_post:
 
             with self.assertRaises(RequestConnectionError):
                 simple_post_with_cached_timeout('abc', 'http://google.com')

